packages/gatsby-plugin-functions/src/gatsby-node.ts

```diff
@@ -129,26 +126,43 @@ export async function onCreateDevServer(
   )
   const files = await glob(functionsGlob, { cwd: functionsDirectory })
 
-  // console.log({ files })
+  reporter.info(`Attaching Functions to development server.`)
 
   const knownFunctions = new Map(
     files.map(file => [
       urlResolve(path.parse(file).dir, path.parse(file).name),
       file,
     ])
   )
-  // console.log(app, functionsDirectoryPath)
 
   app.use(`/api/:functionName`, (req, res, next) => {
     const { functionName } = req.params
 
     if (knownFunctions.has(functionName)) {
-      const fn = require(path.join(
-        functionsDirectory,
-        knownFunctions.get(functionName) as string
-      ))
+      const activity = reporter.activityTimer(
+        `Executing function ${functionName}`
+      )
```

Comment:
Reporter will be unknown inside a cloud-run container, we'll need to replace it with a cloud alternative. For now, I suggest using `console.log`

Reply:
This is the the handler in gatsby develop, in cloud-run we have a different handler to call the function.